go/exorelay/examples/index.go

```diff
@@ -0,0 +1,23 @@
+package exorelay_examples
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Originate/exocom/go/exorelay"
+	"github.com/Originate/exocom/go/structs"
+)
+
+type Example interface {
+	GetReceivedMessages() []structs.Message
+	Setup(exoRelay *exorelay.ExoRelay)
+}
+
+func Get(name string) Example {
+	switch name {
+	case "message-handler":
+		return &MessageHandlerExample{}
```

Comment:
What other types of examples do you foresee being here?

If you do want to keep this I suggest renaming this directory to `test_fixutres` or something because I was thinking these are code examples of how to use exorelay and this seems a bit over-engineered for that